tests/common/mod.rs

```diff
@@ -130,20 +68,20 @@ pub fn keepldr_exec_crate<'a>(
         .time_limit(Duration::from_secs(TIMEOUT_SECS))
         .terminate_for_timeout()
         .wait()
-        .unwrap_or_else(|e| panic!("failed to run `{}`: {:#?}", bin, e))
-        .unwrap_or_else(|| panic!("process `{}` timed out", bin));
+        .unwrap_or_else(|e| panic!("failed to run `{}`: {:#?}", bin.display(), e))
```

Comment:
I'm surprised that explicit .display() is necessary, the `{}` formatter should already be invoking Display::display.

Reply:
```
error[E0277]: `PathBuf` doesn't implement `std::fmt::Display`
  --> tests/common/mod.rs:52:65
   |
52 |         .unwrap_or_else(|e| panic!("failed to run `{}`: {:#?}", bin, e));
   |                                                                 ^^^ `PathBuf` cannot be formatted with the default formatter; call `.display()` on it
   |
   = help: the trait `std::fmt::Display` is not implemented for `PathBuf`
   = note: call `.display()` or `.to_string_lossy()` to safely print paths, as they may contain non-Unicode data
   = note: this error originates in the macro `$crate::const_format_args` (in Nightly builds, run with -Z macro-backtrace for more info)

```